coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/subaward/ProposalBudgetSubAwardController.java

```diff
@@ -199,14 +199,9 @@ public ModelAndView addSubAward(@ModelAttribute("KualiForm") ProposalBudgetForm
 	@Transactional @RequestMapping(params={"methodToCall=deleteLine", "actionParameters[selectedCollectionPath]=budget.budgetSubAwards"})
 	public ModelAndView deleteLine(@RequestParam("actionParameters[lineIndex]") Integer subAwardIndex, @ModelAttribute("KualiForm") ProposalBudgetForm form) {
 		BudgetSubAwards subAwardToDelete = form.getBudget().getBudgetSubAwards().get(subAwardIndex);
-		form.getBudget().getBudgetLineItems().removeAll(subAwardToDelete.getBudgetLineItems());
```

Comment:
@leoherbie, I see that deletematching is removed and currently it is removing line items from period. That is correct.
Are you referring to something different?